requirements.txt

```diff
@@ -14,3 +14,9 @@ statsmodels==0.11.1
 tables==3.6.1
 umap-learn==0.4.6
 xarray==0.12.3
+#Twisted~=20.3.0
```

Comment:
Now that we're pip-installing, I think we can delete these requirements

Reply:
I'm not sure we can delete all these requirements. Twisted is used (for the reactor), and it is not listed in the requirements file of kisok-client. When using twisted, it is necessary to have service_identity as well, and we need pyopenssl to establish the connection (even though the last two modules are not explicitly called).